plugins/main/bridge/bridge.go

```diff
@@ -465,8 +461,35 @@ func cmdAdd(args *skel.CmdArgs) error {
 			if err := ipam.ConfigureIface(args.IfName, result); err != nil {
 				return err
 			}
+			return nil
+		}); err != nil {
+			return err
+		}
+
+		// check bridge port state
```

Comment:
"wait for bridge port state to be Up"

Reply:
nit: should we make this section into to a utility function?